var/key_value_store/config.json

```diff
@@ -1 +1 @@
-{"tva":0.2,"logo":"s:87:\"https:\/\/n7consulting.fr\/themes\/IOTH-bootstrap-basic\/assets\/images\/logo_n7consulting.png\";","adresse":"s:46:\"2 rue Charles Camichel,  31071 Toulouse CEDEX7\";","url":"s:23:\"https:\/\/n7consulting.fr\";","domaineEmailEtu":"s:15:\"etu.enseeiht.fr\";","domaineEmailAncien":"s:18:\"almuni.enseeiht.fr\";","anneCreation":"s:4:\"1977\";","annee1Jeyser":"s:4:\"2015\";","presidentPrenom":"s:7:\"Antoine\";","presidentNom":"s:6:\"Dupond\";","presidentTexte":"s:15:\"Son Pr\u00e9sident,\";","tresorierPrenom":"s:6:\"Thomas\";","tresorierNom":"s:6:\"Dupont\";","tresorierTexte":"s:15:\"Son Tr\u00e9sorier,\";","nom":"s:13:\"N7 Consulting\";","abbr":"s:3:\"N7C\";","email":"s:23:\"contact@n7consulting.fr\";","gaTracking":null,"namingConvention":"s:3:\"nom\";"}
\ No newline at end of file
+{"tva":0.2,"logo":"s:87:\"https:\/\/n7consulting.fr\/themes\/IOTH-bootstrap-basic\/assets\/images\/logo_n7consulting.png\";","adresse":"s:46:\"2 rue Charles Camichel,  31071 Toulouse CEDEX7\";","url":"s:23:\"https:\/\/n7consulting.fr\";","domaineEmailEtu":"s:15:\"etu.enseeiht.fr\";","domaineEmailAncien":"s:18:\"almuni.enseeiht.fr\";","anneeCreation":"s:4:\"1977\";","annee1Jeyser":"s:4:\"2015\";","presidentPrenom":"s:7:\"Antoine\";","presidentNom":"s:6:\"Dupond\";","presidentTexte":"s:15:\"Son Pr\u00e9sident,\";","tresorierPrenom":"s:6:\"Thomas\";","tresorierNom":"s:6:\"Dupont\";","tresorierTexte":"s:15:\"Son Tr\u00e9sorier,\";","nom":"s:13:\"N7 Consulting\";","abbr":"s:3:\"N7C\";","email":"s:23:\"contact@n7consulting.fr\";","gaTracking":null,"namingConvention":"s:3:\"nom\";"}
```

Comment:
Of course.  

That's why it is not the end of my message. I then explain that in order to keep things easy (and cross platform, because a bash script would not necessary work on windows) I kept the file in the index which is the most straight forward solution.